pagopa/io-app
# chore: [IOBP-236] IDPay CIE how it works bottom sheet

## ⚠️ This PR depends on https://github.com/pagopa/io-app-design-system/pull/73 and #4990 

## Short description
This PR adds a custom hook `useIdPayInfoCieBottomSheet` designed to display a bottom sheet containing detailed information about how the CIE payment authorization works for a discount initiative.

## Design reference
[FIGMA](https://www.figma.com/file/SuYjyjZlieQqRS8n1B6qbF/Flussi-App-ID-Pay?type=design&node-id=9611-30793&mode=design&t=mqcM2GWOFRs4RGrR-4)

## List of changes proposed in this pull request
- Created a new custom hook `useIdPayInfoCieBottomSheet` with the basic `FeatureInfo` components
- Upgraded `@pagopa/io-app-design-system` library

## How to test
Actually this is only the definition of the bottomsheet UI, the hook is still not used anywhere but you can try yourself by invoking the `present` method of this custom hook and check the UI.
